CryptoPkg/Library/BaseCryptLib/Pk/CryptPkcs7VerifyBase.c

```diff
@@ -33,6 +33,43 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
   @retval     FALSE         The P7Data was not correctly formatted for processing.
 
 **/
+
+// MU_CHANGE [BEGIN] - TCBZ2539
+// The below code comes from OpenSSL pk7_doit.c
+static int PKCS7_type_is_other(PKCS7 *p7)

```

Comment:
```
STATIC
INTN
PKCS...
```
Also, shouldn't this return a `BOOLEAN`?

Reply:
See below where corthon retracted